### ISSUE TITLE
Search?

### ISSUE THREAD

**Issue by trusktr** [cedx/docgen.js#1]
Are the search and filter features not implemented yet? They don't seem to do anything (yet?).

**Comment by cedx** [comment]
Yes :cry: The project is still in its development phase, and I lack of time to finish it soon. So you will either have to wait a few months (!), or to submit requests to see this project move forward in the right direction.
